cli/tests/e2e/snapshots/test_output/test_debug_experimental_rule/results.txt

```diff
@@ -31,6 +31,10 @@ Done loading local config from rules/experiment/experiment.yaml
 loaded 1 configs in<MASKED>
 running 1 rules from 1 config rules/experiment/experiment.yaml_0
 No .semgrepignore found. Using default .semgrepignore rules. See the docs for the list of default ignores: https://semgrep.dev<MASKED>
+--- semgrep-core stderr ---
+fatal: not a git repository (or any of the parent directories): .git
```

Comment:
Is this addition in the snapshot normal?

Reply:
No, this was not what we wanted. I updated pysemgrep to only dump the contributions if running `semgrep ci`